include/gatekeeper_net.h

```diff
@@ -19,8 +19,19 @@
 #ifndef _GATEKEEPER_NET_H_
 #define _GATEKEEPER_NET_H_
 
+#include <stdint.h>
+
+/* Configuration for the Network. */
+struct net_config {
+	uint16_t	num_rx_queues;
+	uint16_t	num_tx_queues;
+	uint32_t	num_ports;
+};
+
+struct net_config *alloc_and_return_net_conf(void);
+
 int gatekeeper_init_network(void);
 
-void gatekeeper_free_network(void);
+int gatekeeper_free_network(void);
```

Comment:
This functional block is different from other functional block, since there is no ``run_xxx()`` for it. For the network block, the lua script will allocate the ``struct net_config``, and then call the ``gatekeeper_init_network()`` function. However, the ``gatekeeper_free_network()`` will be called by the host program. The host program doesn't have the pointer to the struct allocated by lua script. Thus, we may have to keep a pointer to this struct in the host program.